Rasa_Bot/actions/actions.py

```diff
@@ -29,8 +30,9 @@ def name(self):
         return "action_get_plan_week"
 
     async def run(self, dispatcher, tracker, domain):
-        
-        plan = "Sure, you should do 2 half-hour running sessions. And please read through this psycho-education: www.link-to-psycho-education.nl."
+        age = 30 # We should get this value from a database.
```

Comment:
Better to indicate this is a constant
```suggestion
        AGE = 30 # We should get this value from a database.
```

Reply:
@svenvanderburg thanks for the changes/feedback. I'm not familiar with these conventions. I will try my best to keep these two in mind!

I added the requirement.txt (accidently in the wrong branch at first...). I think the test with the docker file we should do together as I'm not sure how to do this.

Reply:
It's OK that you're not familiar yet. You'll learn along the way! :) Tip: you can use [flake8](https://flake8.pycqa.org/en/latest/) to check whether you follow the style conventions. We will also set that up in Continuous Deployment later.

Reply:
And let's do some pair coding! I'll propose a time slot.